README.md

```diff
@@ -102,17 +102,16 @@ Arguments:
   --label       Label of the new image, defaults to "RUNTIMEJS"
 ```
 ### Environment Variables*
-There is currently no officially supported process for passing environment variables into a runtime instance. There is a work around that will allow the passing of Kernel Command Line arguments instead.
+While there is a [proposal](runtimejs/runtime/issues/134) to add envirionment variables from the command line, there is currently no officially supported process for passing them into a runtime instance. However, there is a work around that will allow the passing of Kernel Command Line arguments instead.
```

Comment:
Gotta use the full URL (`https://github.com/runtimejs/runtime/issues/134`) instead; I tried following the link from your version of the readme and it gave me a 404.